build/deps/deps.jsonc

```diff
@@ -8,6 +8,36 @@
       "repo": "capnproto",
       "branch": "v2",
       "extra_strip_prefix": "/c++"
+    },
+    {
+      "name": "nbytes",
+      "type": "github_release",
+      "owner": "nodejs",
+      "repo": "nbytes",
+      "build_file": "//:build/BUILD.nbytes"
+    },
+    {
+      "name": "ncrypto",
+      "type": "github_tarball",
+      "owner": "nodejs",
+      "repo": "ncrypto",
+      "branch": "main"
+    },
+    {
+      "name": "dragonbox",
```

Comment:
So far, we've been deliberately using the commit taken from the V8 release we use for ancillary V8 dependencies like dragonbox. We could change that, but I don't think it's in scope here.